CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+
+# How to contribute
+
+Thank you for your interest in contributing to a NetPyNE project!. NetPyNE is open source and available at [GitHub repository][NlN-repo].
+
+[NetPyNE](http://netpyne.org/overview.html) is currently being developed and supported by [Neurosin lab](http://neurosimlab.org/) and accepts contributions in the form of bug reports, fixes, feature additions, and documentation improvements (even just typo corrections). The best way to start contributing is by [opening an issue](https://github.com/Neurosim-lab/netpyne/issues) or [fork][GH-fork] the repository on our GitHub page and make a [Pull Request][GH-pr] with your changes.
+
+Users and contributors to NetPyNE are expected to follow our [coding conventions](#coding-conventions).
+
+#### Table Of Contents
+
+- [Overview of contribution process](#overview-of-contribution-process)
+- [Getting started](#getting-started)
+  - [Configuring git](#configuring-git)
+  - [Make a local copy of GitHub repository](#make-a-local-copy-of-github-repository)
+  - [Basic git commands](#basic-git-commands)
+  - [Connecting to GitHub with SSH (optional)](#connecting-to-github-with-ssh-optional)
+- [Opening an issue](#opening-an-issue)
+- [Opening a pull request](#opening-a-pull-request)
+- [Coding conventions](#coding-conventions)
+
+## Overview of contribution process
+> _Hint:_ **Working on your first pull request?** Learn how from this guide in GitHub. [How to Contribute to Open Source Project](https://opensource.guide/how-to-contribute/)
+
+In general you’ll be working with three different copies of the Neurosim-lab/netpyne repository codebase: the official remote copy at https://github.com/Neurosim-lab/netpyne (usually called upstream), your remote [fork][GH-fork] of the upstream repository (similar URL, but with your username in place of Nuerosim-lab, and usually called origin), and the local copy of the codebase on your computer. The typical contribution process is to:
+
+1. synchronize your local copy with `upstream`
+2. make changes to your local copy
+3. [push][GH-push] your changes to `origin` (your remote fork of the `upstream`)
+4. submit a [pull request][GH-prfork] from your fork into `upstream`
+
+The next sections describe this process in more detail.
+
+## Getting started
+> **Note:** If you are new in GitHubs see this [link][GH-start].

```

Comment:
broken link?

Reply:
I put several indirect links, but they are not supported by github .md
I already fix them. :+1: